c_cxx/QNN_EP/QnnCpu/README.md

```diff
@@ -0,0 +1,25 @@
+## About
+- Builds the sample compiled against the ONNX Runtime built with support for Qualcomm® Neural Network (NN) (QNN) Software Development Kit (SDK)
+- The sample uses the QNN EP but runs on QnnCPU.dll (QC)
+- The sample can be compiled and run on an Intel/AMD AMD64 device or QC or other ARM64 device. Does not require a NPU (Neural Processing Unit)
+
+## Prerequisites
+- Windows 11
+- Visual Studio 2022
+- OnnxRuntime ARM Build with QNN support
+    - Either pre-compiled download OR 
+    - Compiled from onnxruntime source with --use_qnn and --qnn_home $(QNN_SDK_ROOT) parameters
```

Comment:
https://onnxruntime.ai/docs/build/eps.html#qnn will be live later today.
so we can point to those build instructions.

Reply:
we will also have nuget package when ort 1.15 is released (ETA within a day)